### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by rishijain** [rishijain/oneplaceaday#54]
What we think should be done:
1. On menu bar, a option 'newsletter'.
2. Clicking on newsletter will open a new page, which will accept email-id.
3. Whenever user signs up for newsletter, he will receive mail every time when a new post is 'published' and a monthly mail with about one post in that mail.

We will need a new table for all users signed up for newsletters. It is not mandatory to sign up on the site for receiving newsletter. Any user can enter their email id and start receiving newsletter.

**Comment by babiracka** [comment]
Do you want a checkbox ("yes, sign me to a newsletter" or something similar) during registration process or only a page described in this issue?

**Comment by rishijain** [comment]
yes. Just a new page which uses the same layout as now. and when user submits their email id, show them a thank you message. 

No checkbox needed while signing up.

**Comment by babiracka** [comment]
Okay, thank you for the answer!
